### ISSUE TITLE
Updated the engine for compatibility with latest love2d version

### ISSUE THREAD

**Issue by WarpspeedSCP** [jacobbrunson/Love-Lights#1]
[Love-Lights-master.zip](https://github.com/jacobbrunson/Love-Lights/files/322340/Love-Lights-master.zip)

please update the git with it.

**Comment by jacobbrunson** [comment]
I realize this is kinda late, but I'll be happy to merge a pull request if you make one.

**Comment by WarpspeedSCP** [comment]
yeah, I'll do it once I figure out how (and clean up my fork). I don't have a computer handy right now.
